finesse/gui/temp_control.py

```diff
@@ -68,7 +69,9 @@ def _create_figure(self) -> None:
         self._ax = {"hot": ax}
         self._canvas = FigureCanvasQTAgg(self._figure)
 
-        self._figure_num_pts = 10
+        poll_interval = 2  # seconds
+        interval_to_show = 15  # minutes
```

Comment:
How about putting these in `config.py` with the other constants?

Reply:
Ah yes, good idea. I was wondering about where to set the poll interval as it belongs to the DP9800 but gets used in various places.

I always find any of the default matplotlib settings triggering, so I'm happy to change the line colours 😅